src/olympia/abuse/models.py

```diff
@@ -74,6 +75,17 @@ class AbuseReport(ModelBase):
         # previous one. We avoid re-using the value.
         ('UNWANTED', 9, "Wasn't wanted / impossible to get rid of"),
         # `10` was previously "Other". We avoid re-using the value.
+
```

Comment:
Black complains about this extra newline here

Reply:
Weird - it doesn't locally for me!